pkg/action/release_testing.go

```diff
@@ -46,7 +46,7 @@ func NewReleaseTesting(cfg *Configuration) *ReleaseTesting {
 }
 
 // Run executes 'helm test' against the given release.
-func (r *ReleaseTesting) Run(name string) (*release.Release, error) {
+func (r *ReleaseTesting) Run(name string, filters map[string][]string) (*release.Release, error) {
```

Comment:
isn't this a breaking change to add a required argument to this method? why not just add it to the `ReleaseTesting` struct?

Reply:
It didn't appear to be a breaking change as I didn't find any other place calling the method. As for why I didn't add it to `ReleaseTesting`, the simple answer is that this is the only Go code I've ever written. I wasn't even aware of the meaning of that method syntax until just now.

Reply:
`Run` is a part of the public API and Helm uses SemVer. That means function signature can't change. It can be documented as deprecated and a new function created and used with the new signature.

I realize this is in a test and it's highly unlikely that this function is in use by anyone. But, there is no way to check that.

This function is also listed on pkg.go.dev.

So, can we do this in a way that doesn't change any signature.

Reply:
Makes sense. I should be able to revisit it this weekend.

Are there any other changes needed?

Reply:
I've made this requested change. My apologies for the delay. Busy times.

Reply:
@seaneagan: For some reason, it won't allow me to request a re-review from you. I'm adding the mention just to be sure you're aware that the change has been made.